.gitignore

```diff
@@ -141,4 +141,4 @@ cython_debug/
 
 .vscode
 .DS_Store
-data/
+data/
```

Comment:
do we use this folder at all?

Reply:
Yes. it's default directory for `biomage experiment pull`